datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/snmp_tile/{check_name}/manifest.json

```diff
@@ -0,0 +1,36 @@
+{{
+  "display_name": "{integration_name}",
+  "maintainer": "{email}",
+  "manifest_version": "1.0.0",
+  "name": "{check_name}",
+  "metric_prefix": "snmp.",
+  "metric_to_check": "",
+  "metric_integration_check_name": "snmp",
```

Comment:
🤔  I thought we are not adding new entry to manifest for now ?

cc @nmuesch

Reply:
probably bad merge, or old bad rebase with previous PR that was editing ddev. Thank you for catching it

Reply:
ah no, I hadn't updated yet, I was waiting for the first branch in the  PRs train to be merged